source/common/logger/ur_sinks.hpp

```diff
@@ -13,6 +13,7 @@ class Sink {
   public:
     template <typename... Args>
     void log(logger::Level level, const char *fmt, Args &&...args) {
+        *ostream << "<" << logger_name << ">";
         *ostream << "[" << level_to_str(level) << "]:";
```

Comment:
I think there should be a space after `:`

Reply:
Done